src/app/_locales/en/quick-start-guide-ios.md

```diff
@@ -0,0 +1,21 @@
+### Welcome to this project, you will need two main steps to make the extension work:
+
+- **Open the App and set a directory**
+
+  - This directory is used to saving and loading your userscripts.
+  - A default local directory is already set when you first launch the App.
+  - Click the current directory to jump to the default directory or the one you set.
+
+- **Enable the extension in Safari**
+
+  - Manage extensions from Settings App (Settings > Safari > Extensions) or Safari App (`AA` button on iPhone, `extension-icon` on iPad, also where you open the extension's `popup` interface)
+  - For optimal experience it's recommended that you "Always Allow" Userscripts for "All Websites", but you could also allow only the sites you want, but please don't forget your settings when your script doesn't work.
+
+### So, how do you install a userscript?
+
+- Visit any `.user.js` URL in Safari, then open the extension popup and you will see an installation prompt, or it will pop up automatically if you have enhanced prompts turned on
+- You can also save `.user.js` files directly to the Userscripts directory you set above, the script file must contain valid metadata block to match the page and be displayed in the extension popup
```

Comment:
```suggestion
### How to you install a user script

- Visit any `.user.js` URL in Safari, then open the extension popup and you will see an installation prompt, or it will pop up automatically if you have enhanced prompts turned on.
- You can also save `.user.js` files directly to the Userscripts directory. The script file must contain a valid metadata block to match the page and be displayed in the extension popup,
```